common/dynamicconfig/constants.go

```diff
@@ -556,6 +556,9 @@ const (
 	// DurableArchivalEnabled is the flag to enable durable archival
 	DurableArchivalEnabled = "history.durableArchivalEnabled"
 
+	// WorkflowExecutionMaxInFlightUpdates is the max number of updates that can be in-flight (admitted but not yet completed)
```

Comment:
maybe specify that this is per-host

Reply:
Its per-workflow-execution. I added to the comment.